administrator/components/com_media/config.xml

```diff
@@ -31,24 +31,6 @@
 		/>
 
 		<field
-			name="file_path"
```

Comment:
Because of backwards compatibility we can't remove the options here. They need to stay as default. It will take at least till Joomla 5 till we can remove the options.

Reply:
is this only for file_path ?

Reply:
No both options.